template/runners_docker_options.tftpl

```diff
@@ -0,0 +1,4 @@
+  [runners.docker]
+%{ for key, value in options ~}
+    ${key} = ${jsonencode(value)}
```

Comment:
@kayman-mk hahaha! 😅 man, look at me doing all fancy for in and you solving it with one single function 🤦🏾

Reply:
Saw it somewhere else. Quite neat!